_docker/drupal/drupal-filesystem/web/sites/example.settings.local.php

```diff
@@ -90,7 +90,7 @@
  * During development it can be useful to install test extensions for debugging
  * purposes.
  */
-$settings['extension_discovery_scan_tests'] = TRUE;
+# $settings['extension_discovery_scan_tests'] = TRUE;
```

Comment:
Should this be part of the commit?

Reply:
It seems to get generated every time I build, but since it doesn't/shouldn't affect anything (I don't think) I figured this way I don't have to worry about it.